birdhouse/deployment/deploy.sh

```diff
@@ -100,56 +99,56 @@ fi
 # Setup COMPOSE_DIR and PWD for sourcing env.local.
 # Prevent un-expected difference when this script is run inside autodeploy
 # container and manually from the host.
-cd $COMPOSE_DIR
+cd "${COMPOSE_DIR}" || exit
 
-START_TIME="`date -Isecond`"
-echo "deploy START_TIME=$START_TIME"
+START_TIME="$(date -Isecond)"
+echo "deploy START_TIME=${START_TIME}"
 
-. "${COMPOSE_DIR}/scripts/read-configs.include.sh"
+. "${COMPOSE_DIR}/read-configs.include.sh"
 
 # Read AUTODEPLOY_EXTRA_REPOS
 read_basic_configs_only
 
 set -x
 
-for adir in $COMPOSE_DIR $AUTODEPLOY_EXTRA_REPOS; do
-    if [ -d "$adir" ]; then
-        cd $adir
+for adir in "${COMPOSE_DIR}" ${AUTODEPLOY_EXTRA_REPOS}; do
+    if [ -d "${adir}" ]; then
+        cd "${adir}" || exit
```

Comment:
No we do not want to `exit` here because we allow non-existing dir in `AUTODEPLOY_EXTRA_REPOS` !

Reply:
see https://github.com/bird-house/birdhouse-deploy/pull/415#discussion_r1497965528